pkg/cnab/config-adapter/testdata/porter-depsv2.yaml

```diff
@@ -0,0 +1,56 @@
+# TODO(PEP003): Get rid of this file, and use porter.yaml with depsv2 enabled instead of having extra bundles
+schemaVersion: 1.0.0-alpha.1
+name: porter-hello
+description: "An example Porter configuration"
+version: 0.1.0
+registry: "localhost:5000"
+
+credentials:
+  - name: username
+    description: Name of the database user
+    required: false
+    env: ROOT_USERNAME
+  - name: password
+    path: /tmp/password
+    applyTo:
+      - uninstall
+
+parameters:
+  - name: db_collation
+    type: string
+    default: magic
+
+dependencies:
+  requires:
+    - name: mysql
+      bundle:
+        reference: "getporter/azure-mysql:5.7"
+      parameters:
+        database: wordpress
+        collation: magic
+      credentials:
+        user: bundle.credentials.username
```

Comment:
I find this super awkward that we don't use template syntax for the wiring. Let's explain why/why not and understand what's possible before locking this in.